### PR TITLE
Add libutil routines to address pid-file generation/handling

### PR DESCRIPTION
**Pull request by rodnymolina**
Signed-off-by: Rodny Molina <rmolina@nestybox.com>

### CODE REVIEW

File: utils/pidfile.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// CreatePidFile writes a sysbox pid to a file. If the file already exists,
+// and its pid matches a current sysbox process, then an error is returned.
+func CreatePidFile(process string, pidFile string) error {
```
Comment (ctalledo) [inline utils/pidfile.go:32]: nit: "process" -> "program" ?

File: utils/pidfile.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+	pid, err := readPidFile(pidFile)
+	if err != nil && !os.IsNotExist(err) {
+		return err
+	}
+
+	if err == nil {
+		if isProcessRunning(process, pid) {
```
Comment (ctalledo) [inline utils/pidfile.go:40]: nit: isProcessRunning -> isProgramRunning ?

**Review by ctalledo — APPROVED** [review]
LG, just a couple of nits.

**Comment by rodnymolina** [comment]
Addressed both review comments.

**Pull request merged**